deploy/docker/pump.io.json

```diff
@@ -0,0 +1,23 @@
+{
+    "driver":  "mongodb",
+    "params":  {"dbname": "pumpio", "host":"127.0.0.1", "port":27017},
+    "secret":  "put a secret string here",
```

Comment:
I don't know offhand, unfortunately. Telling admins to just bring their own pump.io.json seems relatively reasonable to me...